internal/buildscripts/update-deps

```diff
@@ -12,6 +12,20 @@ CONTRIB_VERSION="${CONTRIB_VERSION:-$OTEL_VERSION}"
 
 trap "cd $CUR_DIR" EXIT
 
+PV_RE="v[[:digit:]]+.[[:digit:]]+.[[:digit:]]+-[[:digit:]].([[:digit:]]{14}-[[:alnum:]]{12})"
+
+# account for pseudoversions for pkg/ottl that require v0.0.0 prefix
+desired_contrib_version() {
+    CNTRB_VRSN=$CONTRIB_VERSION
+    if [[ $CONTRIB_VERSION =~ $PV_RE ]]; then
+       PV="${BASH_REMATCH[1]}"
+       if [[ $1 =~ (contrib/pkg/ottl|contrib/pkg/stanza) ]]; then
+           CNTRB_VRSN="v0.0.0-${PV}"
+       fi
+    fi
+		echo $CNTRB_VRSN
```

Comment:
tabs and spaces

Reply:
idea's vim plugin doesn't support my vimrc for some reason.